movePlayerScript.js

```diff
@@ -0,0 +1,14 @@
+var storedKeyPress = 'a';
+
+function moveLeg (keyPress, legChoice) {
+    $(document).keydown(function(e){
+        if(e.keyCode == keyPress && e.keyCode !== storedKeyPress) {
```

Comment:
the key.down event should call the named function not be a part of the function.

Reply:
There is no key.down event, please reread

Reply:
keydown(function(e) { ? but then again, It's really quite arbitrary. I would have said that with this the funciton is doing too much, discreet functions are best.

Reply:
It literally can't do less